### PR TITLE
More human readable dump

### PR DESCRIPTION
**Pull request by RiderExMachina**
I made it so it's in MiB instead of KiB. Makes the game sizes more human readable and understandable. I also tried writing a percentage calculator, but that seems to be having a few minor difficulties. You can see those at 360 of main.cpp

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
+Forked from neobrain. See the original project at [https://github.com/neobrain/braindump](https://github.com/neobrain/braindump)
```
Comment (InusualZ) [inline README.md]: This is not needed since it's the original project

File: source/main.cpp [hunk 2]
```diff
@@ -146,7 +146,7 @@ static std::vector<uint8_t> ReadTitleContent(uint64_t title_id, uint8_t media_ty
         FSFILE_Close(file_handle);
         return {};
     } else {
-        std::cout << (size / 1024) << " KiB... " << std::flush;
+		std::cout << (size / 1024) << " KiB... " << std::flush;
```
Comment (InusualZ) [inline source/main.cpp:149]: You broke the format. Ditto for below

**Review by InusualZ — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (RiderExMachina) [reply README.md]: Heh, yeah. Didn't know how to remove that from the commit

File: source/main.cpp (hunk 2 above)
Comment (RiderExMachina) [reply source/main.cpp:149]: Ah, okay. I'll be more careful in the future

**Comment by neobrain** [comment]
Thanks for the pull request, and sorry for the late reply - this PR slipped somehow slipped my inbox queue and I just accidentally "discovered" it.

Anyway, to get this merged, please:
* Fix the formatting. There are some improper whitespace changes that should be easy to fix by looking at how the existing code is formatted
* Merge your patches into a single one
* Drop the Readme change from this PR. It's unrelated and hence should go into a separate PR (but since it's a simple change, I just edited the Readme manually so you don't need to provide a separate PR in this instance)
* Fix the grammar and spelling mistakes in the code comments

**Pull request closed without merging**